gemfileparser/__init__.py

```diff
@@ -25,65 +27,51 @@
 
 
 class GemfileParser(object):
-
-    '''Creates a GemfileParser object to perform operations. '''
+    """
+    Creates a GemfileParser object to perform operations.
+    """
 
     class Dependency(object):
-
-        ''' A class to hold information about a dependency gem.'''
+        """
+        A class to hold information about a dependency gem.
+        """
 
         def __init__(self):
-            self.name = None
+            self.name = ''
             self.requirement = []
-            self.autorequire = None
-            self.source = None
+            self.autorequire = ''
+            self.source = ''
             self.parent = []
-            self.group = None
-            self.platform = None
-            self.platforms = []
-            self.groups = []
```

Comment:
I guess you removed these attributes because they are either not in the spec or seldom used or parsable?
platform is still a thing though rarely there alright https://guides.rubygems.org/specification-reference/#platform=
Can you confirm?

I never saw a group in a spec so far... did you?

Reply:
I think they are used in `gemfile` files. `gemfileparser` is for both gemspec and gemfile.